ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -153,6 +153,16 @@ tests:
       COMPUTE_NODE_TYPE: c5n.metal
       CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
     workflow: hypershift-kubevirt-e2e
+- as: e2e-kubevirt-aws-csi
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      CNV_PRERELEASE_VERSION: "4.13"
+      COMPUTE_NODE_REPLICAS: "1"
+      COMPUTE_NODE_TYPE: c5n.metal
+      CONTROL_PLANE_INSTANCE_TYPE: m4.xlarge
+    workflow: hypershift-kubevirt-csi-e2e
```

Comment:
I think we want to start by putting this in the kubevirt-csi repo instead of hypershift. 

meaning, in this file. https://github.com/openshift/release/blob/master/ci-operator/config/openshift/kubevirt-csi-driver/openshift-kubevirt-csi-driver-master.yaml

Reply:
Done